### PR TITLE
Newsletters: add 219 (2022-09-28)

### PR DESCRIPTION
**Pull request by harding**
- [x] Lede, releases/RCs, topic links @harding
- [x] Stack Exchange section @bitschmidty

### CODE REVIEW

File: _posts/en/newsletters/2022-09-28-newsletter.md [hunk 1]
```diff
@@ -67,7 +67,30 @@ answers posted since our last update.*
...
+  wallet][topic bip32] is not possible, other onchain data can be used to
+  fingerprint wallet software including type of inputs used, types of outputs
+  created, order of inputs and outputs in the transaction, [coin
+  selection][topic coin selection] algorithm, and use of [timelocks][topic timelocks].
+
+- [Why is there a 5 day gap between the genesis block and block 1?]({{bse}}115344)
+  Murch notes that the gap in the timeline could be explained by the genesis
+  block having a higher difficulty target than required, Satoshi setting the
+  block's timestamp in the past, or the [original Bitcoin software waiting][github
+  jrubin annotated] for a peer before continuing to mine.
```
Comment (Xekyo) [inline _posts/en/newsletters/2022-09-28-newsletter.md:81]:
```suggestion
  jrubin annotated] for a peer before starting to mine.
```

**Review by Xekyo — COMMENTED** [review]
Just did a quick pass

File: _posts/en/newsletters/2022-09-28-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+  forward.  If the peer later attempts to cheat, the honest node can
+  spend the reserve.  Core Lightning defaults to a channel reserve of 1%
+  of the channel balance, penalizing by that amount any peers that
+  attempt to cheat.
+
+    This merged PR allows the user to reduce the reserve for a specific
+    channel, including reducing it to zero.  Although this can be
+    dangerous---the lower the reserve, the less penalty there is for
+    cheating---it can be useful for certain situations.  Most notably,
+    setting the reserve can allow the remote peer to spend all of their
```
Comment (LarryRuane) [inline _posts/en/newsletters/2022-09-28-newsletter.md:126]: (check me on this!)
```suggestion
    setting the reserve to zero can allow the remote peer to spend all of their
```

File: _posts/en/newsletters/2022-09-28-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+  allow a node to advertise four tiered rates for forwarding fees.  For
+  example, a forwarded payment that leaves a channel with less than 25%
+  of its outbound capacity available to forward subsequent payments
+  would need to pay proportionally more than a payment which leaves 75%
+  of its outbound capacity available.
+
+    Developer ZmnSCPxj [described][zmnscpxj ratecards] a simple way to
+    use ratecards, "you can model a rate card as four separate channels
+    between the same two nodes, with different costs each.  If the path
+    at the lowest cost fails, you just try at another route that may
```
Comment (LarryRuane) [inline _posts/en/newsletters/2022-09-28-newsletter.md:25]:
```suggestion
    at the lowest cost fails, you just try another route that may
```

File: _posts/en/newsletters/2022-09-28-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+    have more hops but lower effective cost, or else try the same
+    channel at a higher cost."
+
+    The proposal also allows for negative fees.  For example, a channel
+    could subsidize payments forwarded through it when it had more than
+    75% outbound capacity by adding 1 satoshi to every 1,000 satoshi in
```
Comment (LarryRuane) [inline _posts/en/newsletters/2022-09-28-newsletter.md:31]:
```suggestion
    75% outbound capacity by adding 1 satoshi to every 1,000 satoshis in
```

File: _posts/en/newsletters/2022-09-28-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+    payment value.  Negative fees could be used by merchants to
+    incentivize others to restore inbound capacity to channels
+    frequently used for receiving payments.
+
+    Neigut notes that some nodes are already adjusting their fees based
+    on channel capacity, and that fee ratecards provides a more
```
Comment (LarryRuane) [inline _posts/en/newsletters/2022-09-28-newsletter.md:37]:
```suggestion
    on channel capacity, and that fee ratecards provide a more
```
Or maybe "and that fee ratecards feature provides a more ..."

File: _posts/en/newsletters/2022-09-28-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+    efficient way to communicate their intentions to the network than
+    frequently advertising new feerates via the LN gossip network.
+
+- **Bitcoin implementation designed for testing soft forks on signet:**
+  Anthony Towns [posted][towns bi] to the Bitcoin-Dev mailing list a
+  description of a fork of Bitcoin Core he's working on that will only
+  operate on the default [signet][topic signet] and which will enforce
+  rules for soft fork proposals with high quality specification and
```
Comment (LarryRuane) [inline _posts/en/newsletters/2022-09-28-newsletter.md:45]:
```suggestion
  rules for soft fork proposals with high-quality specifications and
```

File: _posts/en/newsletters/2022-09-28-newsletter.md [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [Is it possible to fingerprint HD wallets?]({{bse}}115311)
+  Pieter Wuille points out that while identifying UTXOs created using an [HD
+  wallet][topic bip32] is not possible, other onchain data can be used to
```
Comment (LarryRuane) [inline _posts/en/newsletters/2022-09-28-newsletter.md:72]:
```suggestion
  wallet][topic bip32] is not possible, other on-chain data can be used to
```

File: _posts/en/newsletters/2022-09-28-newsletter.md [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+  fingerprint wallet software including type of inputs used, types of outputs
```
Comment (LarryRuane) [inline _posts/en/newsletters/2022-09-28-newsletter.md:73]:
```suggestion
  fingerprint wallet software including types of inputs used, types of outputs
```

File: _posts/en/newsletters/2022-09-28-newsletter.md [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+  created, order of inputs and outputs in the transaction, [coin
+  selection][topic coin selection] algorithm, and use of [timelocks][topic timelocks].
+
+- [Why is there a 5 day gap between the genesis block and block 1?]({{bse}}115344)
```
Comment (LarryRuane) [inline _posts/en/newsletters/2022-09-28-newsletter.md:77]:
```suggestion
- [Why is there a 5-day gap between the genesis block and block 1?]({{bse}}115344)
```

File: _posts/en/newsletters/2022-09-28-newsletter.md [hunk 10]
```diff
@@ -0,0 +1,150 @@
...
+  block's timestamp in the past, or the [original Bitcoin software waiting][github
+  jrubin annotated] for a peer before continuing to mine.
+
+- [Is it possible to set RBF as always-on in bitcoind?]({{bse}}115360)
+  Michael Folkson and Murch explain the `walletrbf` configuration option and
+  list a series of related changes involving defaulting to [RBF][topic rbf] in the
+  GUI, defaulting to RBF in RPCs, and using [`mempoolfullrbf`][news208
+  mempoolfullrbf] to allow replacements without signaling.
+
+- [Why would I need to ban peer nodes in Bitcoin network?]({{bse}}115183)
```
Comment (LarryRuane) [inline _posts/en/newsletters/2022-09-28-newsletter.md:89]:
```suggestion
- [Why would I need to ban peer nodes in the Bitcoin network?]({{bse}}115183)
```

File: _posts/en/newsletters/2022-09-28-newsletter.md [hunk 11]
```diff
@@ -0,0 +1,150 @@
...
+  In contrast to [discouraging a peer][bitcoin 23.x banman], user RedGrittyBrick
+  explains that a node operator could choose to manually ban a peer using the
+  [`setban`][setban rpc] RPC if the peer is misbehaving, a suspected malicious or surveillance
+  node, or part of cloud provider's network, among other reasons.
```
Comment (LarryRuane) [inline _posts/en/newsletters/2022-09-28-newsletter.md:93]:
```suggestion
  node, or part of a cloud provider's network, among other reasons.
```

**Review by LarryRuane — COMMENTED** [review]
LGTM, suggesting a few minor grammar tweaks.

File: _posts/en/newsletters/2022-09-28-newsletter.md (hunk 7 above)
Comment (harding) [reply _posts/en/newsletters/2022-09-28-newsletter.md:72]: We previously settled on "onchain", see https://github.com/bitcoinops/bitcoinops.github.io/blob/master/STYLE.md#compound-words

File: _posts/en/newsletters/2022-09-28-newsletter.md [hunk 12]
```diff
@@ -0,0 +1,150 @@
...
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [Is it possible to fingerprint HD wallets?]({{bse}}115311)
+  Pieter Wuille points out that while identifying UTXOs created using an [HD
+  wallet][topic bip32] is not possible, other onchain data can be used to
+  fingerprint wallet software including type of inputs used, types of outputs
+  created, order of inputs and outputs in the transaction, [coin
+  selection][topic coin selection] algorithm, and use of [timelocks][topic timelocks].
```
[the remark this answers is not in the document]
Comment (bitschmidty) [reply _posts/en/newsletters/2022-09-28-newsletter.md:75]: Reworded the question, but took some liberties

**Review by glozow — COMMENTED** [review]
LGTM, awesome newsletter this week

**Comment by renepickhardt** [comment]
Not time sensitive, but today I received a decent answer to a two year old fairly upvoted and until today unanswered question on stack exchange https://bitcoin.stackexchange.com/a/115385/81940 thought I notify you about it.

**Comment by harding** [comment]
Made all suggested edits (thanks for reviews @LarryRuane @Xekyo @glozow ), added releases/RCs, added topic links, and reviewed @bitschmidty's section (thanks!).  Thanks everyone!

(Also upvoted the answer to @renepickhardt's SE question, since it was good, so hopefully it ranks high enough for next month's SE section.)

**Comment by bitschmidty** [comment]
Squashed and merged, thank you @harding @Xekyo @LarryRuane @glozow for contributing and reviewing this week!

**Comment by bitschmidty** [comment]
@renepickhardt thanks for the suggestion for the stack exchange, will consider it for next month!

**Pull request merged**